Prevent *_org.svg files from being generated to avoid plist errors

--- Doxyfile.orig	2019-12-18 22:42:41 UTC
+++ Doxyfile
@@ -1235,7 +1235,7 @@ DOT_IMAGE_FORMAT       = svg
 # The default value is: NO.
 # This tag requires that the tag HAVE_DOT is set to YES.
 
-INTERACTIVE_SVG        = YES
+INTERACTIVE_SVG        = NO
 
 # The tag DOT_PATH can be used to specify the path where the dot tool can be 
 # found. If left blank, it is assumed the dot tool can be found in the path.
